docs/contributing-via-pr.md

```diff
@@ -0,0 +1,42 @@
+# Submitting your Operator via Pull Requests (PR) in community operators project
+
+## Overview
+To submit an operator one has to do these steps
+
+1. Fork project `https://github.com/operator-framework/community-operators`
```

Comment:
This is wrong URL, please replace it with https://github.com/redhat-openshift-ecosystem/community-operators-prod